test/reporter-html/unhandled-rejection.js

```diff
@@ -0,0 +1,78 @@
+// Detect if the current browser supports `onunhandledrejection` (avoiding
+// errors for browsers without the capability)
+var HAS_UNHANDLED_REJECTION_HANDLER = "onunhandledrejection" in window;
+
+if ( HAS_UNHANDLED_REJECTION_HANDLER ) {
+	QUnit.module( "Unhandled Rejections inside test context", function( hooks ) {
+		hooks.beforeEach( function( assert ) {
+			var originalPushResult = assert.pushResult;
+			assert.pushResult = function( resultInfo ) {
+
+			// Inverts the result so we can test failing assertions
```

Comment:
Curious to know why this comment is de-indented from the line below.

Reply:
Just a mistake, though I'm surprised that the otherwise aggressive indentation linting rules didn't catch it...

Reply:
No worries. The indent rule is slightly lenient about comments because in general it is hard to tell what lines of code a comment might be associated with, so we generally allow comments to align with the next line of code, the previous line of code, or with the "correct" indentation (i.e., the indentation we would expect a token to actually be at if this were a token instead of a comment).

My guess is the indent rule is allowing this location due to aligning with line 9. That seems weird since line 10 is whitespace-only, so I wonder if we could improve the rule to not use the previous-line heuristic if the previous token is actually 2 lines above the comment instead of 1 (and similarly, not use the next-line heuristic if the next token is actually 2 lines below the comment instead of 1).

Reply:
Totally makes sense! (also cleaned up the indentation)